app/(auth)/Login.tsx

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import { StyleSheet, Text, View, Image, TouchableOpacity } from "react-native";
+import Input from "@/components/Input";
+import Button from "@/components/Button";
+import ThirdPartyButton from "@/components/ThirdPartyLoginButton";
+import { router } from "expo-router";
+
+const Login = () => {
+  return (
+    <View style={styles.Container}>
+      <Image
+        style={styles.Logo}
+        source={require("../../assets/images/logo.png")}
+      />
+      <Text style={styles.PageTitle}>Login</Text>
+      <View style={styles.Row}>
+        <Text style={styles.Col1}>Don’t have an account?</Text>
+        <TouchableOpacity
+          onPress={() => {
+            router.push({
+              pathname: "/(auth)/Register",
+            });
+          }}
+        >
+          <Text style={styles.Col2}> Register here.</Text>
+        </TouchableOpacity>
```

Comment:
This should be a variant of button component

Reply:
Will change in a latter commit.